source/_patterns/03-uswds/input/input.twig

```diff
@@ -0,0 +1,23 @@
+<div class="usa-form-group {{ group_modifier_classes }}">
+  <label class="usa-label {{ label_modifier_classes }}" for="{{ input_id }}">
+    {{ input_label }}
+    {% if not input_required %}<span class="usa-hint">({{ 'optional'|t }})</span>{% endif %}
+  </label>
+  {% if error_message %}
+    <span class="usa-error-message" id="{{ input_id }}-message" role="alert">{{ error_message }}</span>
+  {% endif %}
+  {% spaceless %}
+  <input class="usa-input {{ modifier_classes }}"
+         id="{{ input_id }}"
+         name="{{ input_id }}"
+         type="{{ input_type|default('text') }}"
+         {% if error_message %} aria-describedby="{{ input_id }}-message"{% endif %}
+         {% if input_size %} size="{{ input_size }}"{% endif %}
+         {% if input_max %} maxlength="{{ input_max }}"{% endif %}
+         {% if input_placeholder %} placeholder="{{ input_placeholder }}"{% endif %}
+         {% if input_title %} title="{{ input_title }}"{% endif %}
+         {% if input_disabled %} disabled{% endif %}
+         {% if input_required %} required{% endif %}
+         {% if input_step %} step="{{ input_step }}"{% endif %}>
```

Comment:
I think lines 11-21 should only be indented 2 spaces instead of seven.

Is there a coding styleguide we’re following that these should be indented to match the first attribute position?

Reply:
That's what PHPStorm defaults to, and I find it easy to read, but there is not a coding styleguide that defines that rule. Is there one that defines 2 spaces?

Reply:
For Gesso (and this fork) we generally follow the Drupal coding standards, which is two spaces for indentation.

Reply:
Works for me. I'll update.